frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx

```diff
@@ -1,15 +1,12 @@
 import * as React from 'react';
 import * as _ from 'lodash';
-import { LoadingInline, pluralize } from '@console/internal/components/utils';
 import { RedExclamationCircleIcon } from '@console/shared';
-
-const pluralizeWithNotation = (i: number, suffix: string, singular: string) =>
-  `${i}${suffix} ${i === 1 ? singular : `${singular}s`}`;
+import { LoadingInline, pluralize, humanizeDecimalBytes } from '@console/internal/components/utils';
 
 const formatCount = (count: number) => {
-  if (count < 1000) return pluralizeWithNotation(count, '', 'Object');
-  if (count >= 1000 && count < 1000000) return pluralizeWithNotation(count / 1000, 'K', 'Object');
-  return pluralizeWithNotation(count / 1000000, 'M', 'Object');
+  const hCount = humanizeDecimalBytes(count);
```

Comment:
We shouldn't be calling `humanizeDecimalBytes` on object counts. These are meant for memory values.

Does `humanizeNumber` work for you?

Reply:
Marking this as resolved...